x-pack/legacy/plugins/canvas/public/components/element_types/index.js

```diff
@@ -15,8 +15,11 @@ import { notify } from '../../lib/notify';
 import { selectToplevelNodes } from '../../state/actions/transient';
 import { insertNodes, addElement } from '../../state/actions/elements';
 import { getSelectedPage } from '../../state/selectors/workpad';
+import { trackCanvasUiMetric } from '../../lib/ui_metric';
 import { ElementTypes as Component } from './element_types';
 
+const customElementAdded = 'custom-element-added';
```

Comment:
nit: consider `elements-custom-added`, depending on if we have any other keys wrt elements.  I notice workpad-related keys start with `workpad-`, which is why I'm suggesting this.

I'd also like to see a `const` file for all of the telemetry keys, with a matching `TS` enum... but that can be a follow-up, if you like.